### ISSUE TITLE
StatusBar stays when a notification is opened

### ISSUE THREAD

**Issue by AlexandreMaistret** [invertase/react-native-firebase#1183]
### Issue

My app is coded to not show the StatusBar at the top of the device : 
 `
<StatusBar
     translucent
     backgroundColor='rgba(0,0,0,0)'
     barStyle='light-content'
     />
`

It works perfectly when I open the app. But however, when the app is opened from a notification sent from firebase cloud functions, the status bar shows up, and stay even if it renders a screen in which `
<StatusBar
     translucent
     backgroundColor='rgba(0,0,0,0)'
     barStyle='light-content'
     />
` is written.

### Environment

1. OS
Android 
Didn't try on IOS

2. React Native version:
0.51.0

3. RNFirebase Version:
4.0.0

**Comment by IljaDaderko** [comment]
@AlexandreMaistret I don't think this is an issue related to react-native-firebase, sounds more like react-native bug.

Also, check where you are rendering `<StatusBar translucent backgroundColor='rgba(0,0,0,0)' barStyle='light-content' />` is that view / element being rendered on the screen that shows up after you open app via notification or is there a chance that opening app through notification skips view / element that sets statusbar style?

**Comment by AlexandreMaistret** [comment]
Ok, I will put the issue on the react-native github, but however, I have an other module which continues to run even if the app is in background or killed, and it shows a notification to say that it is runing , and when I open the app from this notification, the StatusBar is translucent.

I'm rendering <StatusBar translucent backgroundColor='rgba(0,0,0,0)' barStyle='light-content' /> in App.js , with my StackNavigation, so at the very begining of the app launching.

I have tried to put it in a component which mounts later, and it doesn't do anything, exactly the same problem.

**Comment by AlexandreMaistret** [comment]
I've manage to find a begining of a solution. I have set this in my styles.xml : 

    <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
        <item name="android:windowTranslucentStatus">true</item>
        <item name="android:statusBarColor">#00000000</item>
    </style>

The StatusBar is now translucent even if it opened from a notification, but it is not fully transparent just like before, it is dark-transparent.
